massa-protocol-worker/src/protocol_worker.rs

```diff
@@ -344,6 +346,20 @@ impl ProtocolWorker {
         }
     }
 
+    /// Add an list of operations to a buffer for announcement at the next interval,
+    /// or immediately if the buffer is full.
+    async fn note_operations_to_announce(&mut self, operations: &[OperationId]) {
+        // Add the operations to a list for announcement at the next interval.
+        self.operations_to_announce.extend_from_slice(operations);
+
+        // If the buffer is full,
+        // announce operations immediately,
+        // clearing the data at the same time.
+        if self.operations_to_announce.len() > self.config.max_known_ops_size {
```

Comment:
the buffer size should not be max_known_ops_size but another specific config parameter (eg. op_announcement_buffer_size)

Reply:
How about we re-use `operation_batch_buffer_capacity`?

Reply:
The operation asking system and the announcement batching system are separate, independent mechanisms. Let's keep the configs separate for more flexibility

Reply:
Ok, done.